src/control/client/types.go

```diff
@@ -256,3 +257,25 @@ func (c *controllerFactory) create(address string, cfg *security.TransportConfig
 
 	return controller, err
 }
+
+// AccessControlList is a structure for the access control list.
+type AccessControlList struct {
```

Comment:
It's fine to do it this way, particularly if you think you might add more fields, but I just wanted to point out that you could have made this a type alias for []string instead. That would allow you to define the stringer but you wouldn't have to deal with the struct initialization syntax.

e.g. https://play.golang.org/p/k_6dXTaoXqa

Reply:
Indeed - my decision to use a struct was because of an upcoming task to add owner/group to the output. :)